lib/ansible/modules/web_infrastructure/ansible_tower/tower_credential.py

```diff
@@ -190,56 +190,88 @@
     tower_config_file: "~/tower_cli.cfg"
 '''
 
+import os
+
+from ansible.module_utils.ansible_tower import tower_argument_spec, tower_auth_config, tower_check_mode, HAS_TOWER_CLI
+
 try:
-    import os
     import tower_cli
     import tower_cli.utils.exceptions as exc
 
     from tower_cli.conf import settings
-    from ansible.module_utils.ansible_tower import tower_auth_config, tower_check_mode
-
-    HAS_TOWER_CLI = True
 except ImportError:
-    HAS_TOWER_CLI = False
+    pass
```

Comment:
I'm not sure this is right, because I don't think that `ansible.module_utils.ansible_tower.HAS_TOWER_CLI` exists in `stable-2.4`

It looks like you need a mix of the two, something like this:

```diff
+import os
+
+from ansible.module_utils.ansible_tower import tower_argument_spec, tower_auth_config, tower_check_mode
+
```

```diff
 try:
-    import os		
      import tower_cli
      import tower_cli.utils.exceptions as exc
  		  
      from tower_cli.conf import settings
-    from ansible.module_utils.ansible_tower import tower_auth_config, tower_check_mode		
-		
     HAS_TOWER_CLI = True
  except ImportError:
     HAS_TOWER_CLI = False
```